panel/lxqtpanel.cpp

```diff
@@ -160,19 +173,23 @@ LXQtPanel::LXQtPanel(const QString &configGroup, LXQt::Settings *settings, QWidg
     setLayout(lav);
     this->layout()->addWidget(LXQtPanelWidget);
 
+    //Create, initialize and connect the LXQtPanelLayout.
```

Comment:
I'm -1 on all such needless comments. If the code is selfexplanatory and the comment just repeats what is expressed by the code, then the comment is useless.